odl/util/graphics.py

```diff
@@ -436,7 +436,7 @@ def show_discrete_data(values, grid, title=None, method='',
                 plt.colorbar(mappable=csub, ticks=ticks, format=fmt)
             elif update_in_place:
                 # If it exists and we should update it
-                csub.colorbar.set_clim(minval, maxval)
```

Comment:
Should not be in this PR, this is subject of https://github.com/odlgroup/odl/pull/1638